src/sumOfOther.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function sumOfOther(array){
+ 
+    let result = [];
+    let sum = 0;
+ 
+    for(let i = 0; i < array.length; i++){
+        sum += array[i];
+    }
+    for(let j = 0; j < array.length; j++){
+       let k = sum - array[j];
```

Comment:
Incorrect indents. 
`    let k = sum - array[j];` should be shifted right to 1 space